HealthMonitoring.SelfHost/Content/Scripts/homeController.js

```diff
@@ -23,29 +26,68 @@
         var configFrequency = getConfigUpdatingFrequency($location);
 
         $scope.displayHint = function (evnt) {
-            if ($scope.statLegend == null) {
+            if ($scope.statLegend === null) {
                 $scope.statLegend = {
                     status: evnt.target.innerHTML,
                     description: statusLegend[evnt.target.innerHTML],
                     x: evnt.clientX + 5,
                     y: evnt.clientY + 5
                 };
             }
-        }
+        };
+
+        $scope.getGroupColour = function (group) {
```

Comment:
Who prioritized statuses? bc its interesting that faulty has less priority than timedOut
also can this logic be done in more elegant way without  7 if blocks ?